### ISSUE TITLE
Uncaught TypeError: Object.observe is not a function

### ISSUE THREAD

**Issue by proteo** [fentas/atom-to-base64#4]
This happens on the latest 1.12 beta (5) when using the "To Base64" contextual menu in the tree view.

Steps to reproduce:

1. Select a SVG file on the tree view.
2. Right click and select "To Base64".

Manually selecting file contents and selecting To Base64 > Encode works fine though.

**Atom Version**: 1.12.0-beta5
**Electron Version**: 1.3.6
**System**: Mac OS X 10.11.6
**Thrown From**: [to-base64](https://github.com/fentas/atom-to-base64) package, v0.4.2

### Stack Trace

Uncaught TypeError: Object.observe is not a function

```
At /Users/proteo/.atom/packages/to-base64/lib/to-base64-core.coffee:43

TypeError: Object.observe is not a function
    at /Users/proteo/.atom/packages/to-base64/lib/to-base64-core.coffee:43:14
```

### Commands

```
     -5:01.1.0 editor:move-to-first-character-of-line (atom-text-editor.editor.is-focused)
     -5:00.9.0 editor:select-to-end-of-line (atom-text-editor.editor.is-focused)
     -5:00.8.0 core:cut (atom-text-editor.editor.is-focused)
  2x -4:59 core:paste (atom-text-editor.editor.is-focused)
     -4:52 core:backspace (atom-text-editor.editor.is-focused)
     -4:48.8.0 autocomplete-plus:cancel (atom-text-editor.editor.is-focused.autocomplete-active)
  2x -4:47.9.0 core:save (atom-text-editor.editor.is-focused)
     -4:04.2.0 core:paste (atom-text-editor.editor.is-focused)
  2x -4:03.2.0 core:backspace (atom-text-editor.editor.is-focused)
     -4:02.5.0 core:undo (atom-text-editor.editor.is-focused)
     -4:01.7.0 core:backspace (atom-text-editor.editor.is-focused)
     -4:00.5.0 core:save (atom-text-editor.editor.is-focused)
     -1:00.6.0 core:delete (atom-text-editor.editor.is-focused)
     -0:59.6.0 core:save (atom-text-editor.editor.is-focused)
     -0:20.3.0 core:close (atom-text-editor.editor.is-focused)
     -0:00.3.0 to-base64:view (span.name.icon.icon-file-text)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "metrics",
      "autosave",
      "background-tips",
      "exception-reporting",
      "language-c",
      "language-clojure",
      "language-csharp",
      "language-go",
      "language-mustache",
      "language-objective-c",
      "language-ruby-on-rails",
      "language-ruby",
      "language-python",
      "language-perl",
      "language-make",
      "language-java",
      "language-toml",
      "open-on-github",
      "spell-check",
      "welcome",
      "language-gfm",
      "symbols-view",
      "deprecation-cop",
      "autocomplete-snippets"
    ],
    "ignoredNames": [
      ".git",
      ".hg",
      ".svn",
      "config.codekit",
      ".vscode",
      "*.sublime-project",
      "*.sublime-workspace",
      ".DS_Store",
      "._*",
      "Thumbs.db"
    ],
    "openEmptyEditorOnStart": false,
    "packagesWithKeymapsDisabled": [
      "tool-bar"
    ],
    "projectHome": "/Users/proteo/Sites",
    "telemetryConsent": "no",
    "themes": [
      "one-dark-ui",
      "proteo-dark-theme"
    ],
    "useCustomTitleBar": true,
    "warnOnLargeFileLimit": 5
  }
}
```

### Installed Packages

```coffee
# User
atom-beautify, v0.29.13 (active)
atom-drupal-api, v1.0.1 (inactive)
atom-icons, v0.2.2 (active)
atom-minify, v0.7.5 (active)
autocomplete-php, v0.3.7 (active)
block-comment-plus, v0.4.0 (active)
caniuse, v0.9.0 (active)
custom-title, v1.0.1 (active)
docblockr, v0.8.7 (active)
docker, v0.7.0 (inactive)
drupal, v1.8.0 (active)
editorconfig, v1.7.2 (active)
escape-utils, v0.5.3 (inactive)
fold-lines, v0.2.1 (active)
git-control, v0.8.2 (inactive)
highlight-selected, v0.11.2 (active)
language-docker, v1.1.7 (active)
linter, v1.11.18 (active)
linter-drupalcs, v1.4.1 (active)
linter-less, v2.6.0 (active)
linter-php, v1.3.1 (active)
linter-sass-lint, v1.7.1 (active)
native-color-picker, v0.2.0 (inactive)
php-integrator-autocomplete-plus, v1.1.5 (active)
php-integrator-base, v1.2.6 (active)
php-integrator-linter, v1.2.1 (active)
php-integrator-navigation, v1.0.3 (active)
php-integrator-tooltips, v1.0.4 (active)
pigments, v0.37.0 (active)
project-manager, v3.3.2 (active)
proteo-dark-theme, v1.0.0 (active)
reference-theme, v0.0.0 (inactive)
remember-folds, v0.2.1 (active)
split-diff, v1.0.6 (inactive)
terminal-plus, v0.14.5 (active)
to-base64, v0.4.2 (active)
tool-bar, v1.0.1 (active)
tool-bar-config, v0.1.0 (active)
atom-dark-syntax, v0.27.0 (inactive)
atom-dark-ui, v0.52.0 (inactive)
atom-light-syntax, v0.28.0 (inactive)
atom-light-ui, v0.45.0 (inactive)
base16-tomorrow-dark-theme, v1.3.0 (inactive)
base16-tomorrow-light-theme, v1.3.0 (inactive)
one-dark-ui, v1.6.2 (active)
one-light-ui, v1.6.2 (inactive)
one-dark-syntax, v1.5.0 (inactive)
one-light-syntax, v1.5.0 (inactive)
solarized-dark-syntax, v1.0.5 (inactive)
solarized-light-syntax, v1.0.5 (inactive)
about, v1.7.0 (active)
archive-view, v0.62.0 (active)
autocomplete-atom-api, v0.10.0 (active)
autocomplete-css, v0.13.1 (active)
autocomplete-html, v0.7.2 (active)
autocomplete-plus, v2.31.4 (active)
autocomplete-snippets, v1.11.0 (inactive)
autoflow, v0.27.0 (inactive)
autosave, v0.23.1 (inactive)
background-tips, v0.26.1 (inactive)
bookmarks, v0.42.0 (active)
bracket-matcher, v0.82.2 (active)
command-palette, v0.39.0 (inactive)
deprecation-cop, v0.54.1 (inactive)
dev-live-reload, v0.47.0 (active)
encoding-selector, v0.22.0 (active)
exception-reporting, v0.40.0 (inactive)
find-and-replace, v0.202.0 (active)
fuzzy-finder, v1.4.0 (active)
git-diff, v1.1.0 (active)
go-to-line, v0.31.0 (inactive)
grammar-selector, v0.48.2 (active)
image-view, v0.60.0 (active)
incompatible-packages, v0.26.1 (active)
keybinding-resolver, v0.35.0 (active)
line-ending-selector, v0.5.0 (active)
link, v0.31.2 (inactive)
markdown-preview, v0.158.8 (active)
metrics, v1.0.0 (inactive)
notifications, v0.65.1 (active)
open-on-github, v1.2.1 (inactive)
package-generator, v1.0.1 (inactive)
settings-view, v0.243.1 (active)
snippets, v1.0.3 (active)
spell-check, v0.68.4 (inactive)
status-bar, v1.4.1 (active)
styleguide, v0.47.2 (active)
symbols-view, v0.113.1 (inactive)
tabs, v0.102.2 (active)
timecop, v0.33.2 (active)
tree-view, v0.210.0 (active)
update-package-dependencies, v0.10.0 (active)
welcome, v0.35.1 (inactive)
whitespace, v0.35.0 (active)
wrap-guide, v0.38.2 (active)
language-c, v0.54.0 (inactive)
language-clojure, v0.22.1 (inactive)
language-coffee-script, v0.48.0 (active)
language-csharp, v0.12.1 (inactive)
language-css, v0.40.1 (active)
language-gfm, v0.88.0 (inactive)
language-git, v0.15.0 (active)
language-go, v0.43.0 (inactive)
language-html, v0.46.1 (active)
language-hyperlink, v0.16.1 (active)
language-java, v0.24.0 (inactive)
language-javascript, v0.122.0 (active)
language-json, v0.18.3 (active)
language-less, v0.29.6 (active)
language-make, v0.22.2 (inactive)
language-mustache, v0.13.0 (inactive)
language-objective-c, v0.15.1 (inactive)
language-perl, v0.37.0 (inactive)
language-php, v0.37.3 (active)
language-property-list, v0.8.0 (active)
language-python, v0.45.1 (inactive)
language-ruby, v0.70.2 (inactive)
language-ruby-on-rails, v0.25.1 (inactive)
language-sass, v0.57.0 (active)
language-shellscript, v0.23.0 (active)
language-source, v0.9.0 (active)
language-sql, v0.25.0 (active)
language-text, v0.7.1 (active)
language-todo, v0.29.1 (active)
language-toml, v0.18.1 (inactive)
language-xml, v0.34.12 (active)
language-yaml, v0.27.1 (active)

# Dev
No dev packages
```

**Comment by fentas** [comment]
Thanks for issue notification.
Should be fixed in the v0.5.0.

If not, feel free to reopen.

**Comment by proteo** [comment]
Thanks @fentas, I can confirm it's fixed in 0.5.
